tenders/common/block_attach.c

```diff
@@ -42,9 +44,6 @@ int block_attach(const char *path, off_t *capacity_)
     off_t capacity = lseek(fd, 0, SEEK_END);
     if (capacity == -1)
         err(1, "%s: Could not determine capacity", path);
-    if (capacity < 512)
-        errx(1, "%s: Backing storage must be at least 1 block (512 bytes) "
-                "in size", path);
```

Comment:
Why this line was deleted? According to what you say, we expect a block size equal or greater than 512.

Reply:
Ok, I see. Thanks!

Reply:
We could keep the check in `block_attach()` and report `%s: Backing storage must be at least 1 block (>= 512 bytes) in size`, but then we'd have to check twice; once again when we know the block size.

The problem is if we set block size to say 1024 bytes and the file is 512 bytes - then we can't read or write to the file, but it passes this check. So we have to check (again) once we know the block size.